spec/entry_spec.rb

```diff
@@ -54,10 +54,28 @@
     it 'can handle multiple locales' do
       vcr('entry_locales') {
         cat = create_client.entries(locale: "*").items.first
-        expect(cat.fields[:name][:'en-US']).to eq "Nyan Cat"
-        expect(cat.fields[:name][:'es']).to eq "Gato Nyan"
+        expect(cat.fields('en-US')[:name]).to eq "Nyan Cat"
+        expect(cat.fields('es')[:name]).to eq "Gato Nyan"
```

Comment:
This is actually the expected behavior of `fields`, I was omitting that behavior in order to be able to serialize correctly in `contentful_middleman`.

This, though breaking is undocumented anywhere else than this test, and is fixing the dispaired behavior.

`fields_with_locales` is the in place replacement for this functionality, and it is documented properly